lib/widgets/orderwidget.dart

```diff
@@ -71,14 +99,16 @@ class _OrderWidgetState extends State<OrderWidget> {
         type: StepperType.horizontal,
         physics: NeverScrollableScrollPhysics(),
         controlsBuilder: (BuildContext context,
-            {VoidCallback onStepContinue, VoidCallback onStepCancel}) =>
-            Container(height: 0.0,),
+                {VoidCallback onStepContinue, VoidCallback onStepCancel}) =>
+            Container(
+          height: 0.0,
+        ),
```

Comment:
can you explain why did you change this?

Reply:
it was changed unintentionally due to auto formater of vs code I have made it as it is of master now.

Reply:
great